doc/FAQ.md

```diff
@@ -65,6 +65,29 @@ only to HTTP requests.
 You can use [Let's Encrypt](https://letsencrypt.org/) to get an SSL certificate
 for free.
 
+## How do I access web services?
+
+code-server is capable of proxying to any port using either a subdomain or a
+subpath.
```

Comment:
Not sure that this sufficiently explains the motivations behind this, namely keeping other ports private/hidden. Might make it harder for someone to understand why this is useful given that most setups at the moment likely expose all ports. What about something like: 

```
## How do I keep other web services on my server password protected?
```

Reply:
That makes sense. How do you feel about:

> How do I securely access web services?

And then below that:

> code-server is capable of proxying to any port using either a subdomain or a path which means you can access these services using code-server's authentication.

Reply:
Yeah that looks better to me 👍